app/views/dogs/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Dogs</h1>
+
+<ul>
```

Comment:
put this in a div with a a tag to rent. something like this:
<% @dogs.each do |dog| %>
<div>
</div>
<div class="row">
  <div class="col-sm-6">
    <div class="card">
      <div class="card-body">
        <h5 class="card-title"><%= dog.name %></h5>
        <p class="card-text"><%= dog.description %></p>
        <p class="card-text"><%= dog.price %></p>
      </div>
    </div>
  </div>
  </div>
<% end %>

Reply:
I fixed the code as you suggested along with the titles with the <strong> tags.